collector/README.md

```diff
@@ -61,93 +61,96 @@ marked with a '?' in the `compare` page.
 
 ### How to benchmark a change on your own machine
 
-To benchmark a local build:
+The following command runs the benchmark suite using a local rustc:
 ```
-./target/release/collector --db $DATABASE \
-    bench_local --rustc $RUSTC --cargo $CARGO $ID
+./target/release/collector bench_local <RUSTC> <ID>
 ```
 
-`$DATABASE` is a path (relative or absolute) to a sqlite database file, in which
-the timing data will be placed. It will be created if it does not already exist.
-Alternatively, the collector supports postgres as a backend and the URL can be
-specified (beginning with `postgres://`), but this is unlikely to be useful for
-local collection.
+It will benchmark the entire suite and put the results in a file called
+`results.db`. The full benchmark suite takes some time to run: tens of minutes
```

Comment:
Tens of minutes seems optimistic, I'd expect on "average" hardware for a developer we'd expect runtime in the hours range.